packages/manager/modules/pci/src/projects/project/storages/containers/container-object.class.js

```diff
@@ -0,0 +1,16 @@
+import moment from 'moment';
+
+export default class ContainerObject {
+  constructor(resource) {
+    this.retrievalDate = null;
+    Object.assign(this, resource);
```

Comment:
This looks confusing, don't you think that you could object spread the parameter instead ?

```js
constructor({ retrievalDelay }) 
```

or else I don't know what I'm suppose to use to build a `ContainerObject` :)